website/docs/docs/deploy/job-scheduler.md

```diff
@@ -82,6 +83,18 @@ The scheduler prevents queue clog by canceling runs that aren't needed, ensuring
 
 To prevent over-scheduling, users will need to take action by either refactoring the job so it runs faster or modifying its [schedule](/docs/deploy/deploy-jobs#schedule-days).
 
+## Deactivation of jobs <Lifecycle status='beta' />
+
+To reduce unnecessary resource consumption and reduce contention for run slots in your account, dbt Cloud will deactivate a [deploy job](/docs/deploy/deploy-jobs) or a [CI job](/docs/deploy/ci-jobs) if it reaches 100 Git clone errors and indicate this through the use of banners. When this happens, scheduled and triggered-to-run jobs will no longer be enqueued. 
+
+To reactivate a deactivated job, you can either:
+- Perform a manual run by clicking **Run now** on the job's page
+- Update the job's settings
```

Comment:
Ideally, we want people to fix their jobs (they have been failing 100+ times in a row. Could we generalize this more and reverse the order maybe to say:
- Update the job's settings to fix the issue and save the job 
- Perform a manual run by clicking **Run now** on the job's page